src/converters/array-converter.js

```diff
@@ -1,6 +1,26 @@
+/**
+ * ## Array Converters
+ * 
+ * Converters allow you to convert data -- in particular, model variables that you display in your project's user interface -- from one form to another.
+ *
+ * There are two ways to specify conversion or formatting for the display output of a particular model variable:
+ *
+ * * Add the attribute `data-f-convert` to any element that also has the `data-f-bind` or `data-f-foreach`.
+ * * Use the `|` (pipe) character within the value of any `data-f-` attribute (not just `data-f-bind` or `data-f-foreach`).
+ *
+ * In general, if the model variable is an array, the converter is applied to each element of the array. There are a few built in array converters which, rather than converting all elements of an array, select particular elements from within the array or otherwise treat array variables specially.
+ * 
+ */
+
+
 'use strict';
 var list = [
     {
+        /**
+         * Convert the array into a list by concatenating all elements of the array.
```

Comment:
Convert *input* into an array. i.e. if your variable returns "1" and you want to convert that to ["1"] (maybe to pass in to other converters down the chain which only take in arrays